src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -199,7 +199,7 @@ FieldSpec generateCreator(ProcessingEnvironment env, List<Property> properties,
     return FieldSpec
         .builder(creatorOfClass, "CREATOR", Modifier.PUBLIC,
             Modifier.FINAL, Modifier.STATIC)
-        .initializer(creatorImpl.toString())
+        .initializer("$L", creatorImpl)
```

Comment:
This fixes the problem where code was emitted with fully-qualified types instead of participating in import resolution.